### PR TITLE
Release/v0.0.28

### PR DESCRIPTION
**Pull request by jasonbahl**
- Updates to the main README to make it easier to find some resources about plugin usage
- Introduce `graphql_access_control_allow_headers` filter
- Adjust some sanitization re: GET request handling
- Fixes a bug with undefined variables for GET requests
- Adds `stati` arg to PostObjectConnectionArgs
- Fixes bug re: ordering by `in`, `notIn` and `parentIn`

### CODE REVIEW

File: tests/functional/PostListWithDifferentStatusCept.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	$I->havePostInDatabase( [
+		'post_type'    => 'post',
+		'post_status'  => $post_status,
+		'post_title'   => "test {$post_status} post",
+		'post_content' => "test {$post_status} content"
+	] );
+}
+
+/**
+ * Set the content-type so we get only posts with status published and draft.
```
Comment (chriszarate) [inline tests/functional/PostListWithDifferentStatusCept.php]: comment makes it sound like content-type is at play

Comment (chriszarate) [reply tests/functional/PostListWithDifferentStatusCept.php]: https://github.com/wp-graphql/wp-graphql/pull/439

File: src/Type/PostObject/Connection/PostObjectConnectionArgs.php [hunk 2]
```diff
@@ -243,6 +243,13 @@ private static function fields( $connection ) {
 					'type' => Types::post_status_enum(),
 				],
 
+				/**
+				 * List of post status parameters
+				 */
+				'stati' => [
```
Comment (chriszarate) [inline src/Type/PostObject/Connection/PostObjectConnectionArgs.php]: I understand the logic here to mirror WP_Query functionality. I missed the initial PR; my only question is whether we considered consolidating on a single arg that always takes an array of statuses, even if you are only querying one, e.g.:

```
status: ['draft']
```

```
status: ['draft', 'publish']
```

Comment (jasonbahl) [reply src/Type/PostObject/Connection/PostObjectConnectionArgs.php]: @chriszarate right now the `status` arg accepts an individual status and this introduces a plural `stati` which accepts a list, and does so in a non-breaking fashion.

File: tests/functional/PostListWithDifferentStatusCept.php (hunk 1 above)
Comment (jasonbahl) [reply tests/functional/PostListWithDifferentStatusCept.php]: Thanks for fixing

File: src/Type/PostObject/Connection/PostObjectConnectionArgs.php (hunk 2 above)
Comment (chriszarate) [reply src/Type/PostObject/Connection/PostObjectConnectionArgs.php]: Makes sense, just raising opportunities for consolidation. Glad to have this functionality!

**Pull request merged**